crowbar_framework/app/models/service_object.rb

```diff
@@ -919,6 +919,35 @@ def self.proposal_to_role(proposal, bc_name)
     RoleObject.new role
   end
 
+  #  we can speed-up the application of (n+1)th role if both(n,n+1)
```

Comment:
Thanks for this comment block - definitely important to have it since this batch merging is not immediately obvious without explanation!